pyta.py

```diff
@@ -14,11 +14,17 @@
 
 import webbrowser
 import os
+import sys
 from urllib.request import pathname2url
 
 # Local version of website; will be updated later.
 HELP_URL = 'file:' + pathname2url(os.path.abspath('website/index.html'))
 
+# check the python version
+if sys.version_info < (3, 4, 0):
+    print('You need Python 3.4 or later to run this script\n')
+    exit(1)
```

Comment:
@Jasminewwwu I realized that there shouldn't actually be an `exit` here. Just the warning message is enough.